### ISSUE TITLE
Dynamic scores that depend on all submissions

### ISSUE THREAD

**Issue by mariusbrinkmann** [codalab/codalab-competitions#3129]
Is there a way to compute dynamic scores that depend on all submissions and change once new submissions have been uploaded?

I would like to include the execution time in the scoring. However, instead of using absolute values, e.g., <10 s and <100s, I would like to make multiple categories, e.g. top 10%, top 30%, and top 50%, i.e., the score would depend on all submissions.

**Comment by Didayolo** [comment]
I don't think it is possible, however the participants are **ranked** by the different scores. So knowing e.g. the top 10% should be straightforward.
